### PR TITLE
fix/configure bitmart to estimate balances

### LINKED ISSUE #4115: Bitmart: Random inconsistent order creation due to balance delay
**Describe the bug**
Opening ticket for tracking:
- There are random instances that the order creation is inconsistent. The possible reason for this is due to Bitmart not having websocket update for balance as mentioned in PR #3856

**Steps To Reproduce**
Here are the steps to reproduce the issue (see attachments in the section below):
1. Create a pure market making strategy and set order refresh time to 30 sec for faster result.
2. Set a multiple order level, single level will do but may take more time.
3. Start the strategy and run `status --live`
4. Monitor the order creation.
...

**Screenshots**

https://user-images.githubusercontent.com/57189990/132447195-6d9bc81e-6e17-4af0-aaee-199f93601aba.mov

**Release version**
0.44.0

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**

### PR DESCRIPTION
**Pull request by aarmoa**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:
Changes to make the Bitmart connector estimate balances because the exchange does not provide online balance updates

**Tests performed by the developer**:
All unit tests passing in green

**Tips for QA testing**:
Check balance is kept consistent

Fixes #4115

### CODE REVIEW

**Comment by aarmoa** [comment]
PR marked as draft until #4356 is merged into development branch

File: hummingbot/connector/exchange/bitmart/bitmart_exchange.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
+import aiohttp
+import asyncio
+import copy
+from decimal import Decimal
+import json
```
Comment (CrimsonJacket) [inline hummingbot/connector/exchange/bitmart/bitmart_exchange.py:5]: Minor comment on the import order.

**Review by CrimsonJacket — CHANGES_REQUESTED** [review]
Just a minor comment on the order of imports. Rest LGTM

**Review by RC-13 — APPROVED** [review]
Run a test bot for almost an hour and monitor the status and logs, confirmed that there was no inconsistency with the order creation. LGTM!

**Review by CrimsonJacket — APPROVED** [review]
Changes LGTM.

**Pull request merged**